### PR TITLE
Document that print doesn't work in Python models

### PR DESCRIPTION
**Pull request by joellabes**
## What are you changing in this pull request and why?
Python models don't support print(), because of where the processing happens. 

This is maybe not the most precise documentation I've ever written in my life - someone who is more in the weeds on how Python models work might want to tighten it up a bit and provide any available workarounds. 

Instigating thread: https://discourse.getdbt.com/t/not-able-to-print-data-to-terminal-using-print-statement-in-dbt-core/8092/1

### CODE REVIEW

File: website/docs/docs/build/python-models.md [hunk 1]
```diff
@@ -560,6 +560,7 @@ Python models have capabilities that SQL models do not. They also have some draw
...
 - **These capabilities are very new.** As data warehouses develop new features, we expect them to offer cheaper, faster, and more intuitive mechanisms for deploying Python transformations. **We reserve the right to change the underlying implementation for executing Python models in future releases.** Our commitment to you is around the code in your model `.py` files, following the documented capabilities and guidance we're providing here.
+- **Lack of print() support.** Because your Python model is compiled and run entirely inside the data platform without oversight from dbt, the output of commands like `print()` is not shown in dbt's logs.
```
Comment (mirnawong1) [inline website/docs/docs/build/python-models.md]:
```suggestion
- **Lack of `print()` support.** The data platform runs and compiles your Python model without dbt's oversight. This means it doesn't display the output of commands such as `print()` in dbt's logs. 
```

Comment (mirnawong1) [inline website/docs/docs/build/python-models.md]: sugg active voice

```suggestion
- **Lack of `print()` support.** The data platform runs and compiles your Python model without dbt's oversight. This means it doesn't display the output of commands such as `print()` in dbt's logs. 
```

**Comment by mirnawong1** [comment]
thanks @joellabes , left a small sugg. also i checked for [workarounds](https://dbt-labs.slack.com/archives/C017GDLAF7D/p1683749776239499) in dbt help but i _don't_ think they're ones i'd like to recommend on the docs.

**Comment by mirnawong1** [comment]
Hey @joellabes do you want me to commit and merge this for you?

**Comment by joellabes** [comment]
Thank you! It looks like I dismissed your review by accepting your suggestion, so yes feel free to merge it once the build completes again 🙏

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**